File: app/src/main/java/com/appttude/h_mal/atlas_weather/legacy/AppWidget/WidgetRemoteViewsService.java
package com.appttude.h_mal.atlas_weather.legacy.AppWidget;

import android.content.Intent;
import android.widget.RemoteViewsService;


public class WidgetRemoteViewsService extends RemoteViewsService{

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new MyWidgetRemoteViewsFactory(getApplicationContext(),intent);
    }
}

File: app/src/monoWeather/java/com/appttude/h_mal/atlas_weather/monoWeather/widget/WidgetRemoteViewsService.kt
package com.appttude.h_mal.atlas_weather.monoWeather.widget

import android.content.Intent
import android.widget.RemoteViewsService

class WidgetRemoteViewsService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
        return MyWidgetRemoteViewsFactory(applicationContext, intent)
    }
}